github/ci/prow-deploy/files/jobs/nmstate/kubernetes-nmstate/kubernetes-nmstate-presubmits.yaml

```diff
@@ -115,7 +115,7 @@ presubmits:
         nodeSelector:
           type: bare-metal-external
         containers:
-          - image: kubevirtci/bootstrap:v20201119-a5880e0
+          - image: quay.io/kubevirtci/bootstrap:v20210311-09ebaa2
```

Comment:
flaky here
https://prow.apps.ovirt.org/view/gcs/kubevirt-prow/pr-logs/pull/kubevirt_project-infra/1046/rehearsal-pull-kubernetes-nmstate-e2e-handler-k8s-parallel/1373969280038604800

not sure if due to this change, but reverting from this PR